docs/aqua-book/language/types.md

```diff
@@ -2,23 +2,52 @@
 
 ## Scalars
 
-Scalar types follow the Wasm IT notation.
+Scalar types in Aqua follow the Wasm IT notation.
 
 * Unsigned numbers: `u8`, `u16`, `u32`, `u64`
 * Signed numbers: `i8`, `i16`, `i32`, `i64`
 * Floats: `f32`, `f64`
 * Boolean: `bool`
 * String: `string`
-* Structures (product types): see [Structures](types.md#structures) below
-* Arrays: see [Collection Types](types.md#collection-types) below
```

Comment:
Structures and arrays are also WIT scalars

Reply:
Could you give a reference, please?